pages/basics/index.js

```diff
@@ -10,6 +12,15 @@ function Basics({url, content, locale}) {
       locale={locale}
       contributors={content.contributors}
     >
+      <Head>
+        {twitterCard({
+          card: 'summary',
+          title: content.title,
+          description: content.heading,
```

Comment:
It came across my mind too if the line break would affect the way it displays on Twitter. So far, it looks fine based on how it's being displayed.

Here's the [twitter card `summary`](https://twitter.com/fancyfallancy/status/878929622645194752) look like. 

The other way that I could think of would be to have it's own content in the `content` folder, but not sure if that would be the way to go for now. 😅